### PR TITLE
refactored patWaitingRoom.html

### PR DESCRIPTION
**Pull request by briosolivares**

### CODE REVIEW

File: templates/patWaitingRoom.html [hunk 1]
```diff
@@ -21,12 +21,12 @@
     <script defer src="/__/firebase/init.js"></script> 
 
 
-    <link rel="stylesheet" type="text/css" href="prWaitingRoom.css">
-  <link rel="stylesheet" type="text/css" href="patWaitingRoom.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/bootstrap.min.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/font-awesome.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/templatemo-art-factory.css">
-    <link rel="stylesheet" type="text/css" href="/temp/assets/css/owl-carousel.css">
+    <link rel="stylesheet" type="text/css" href="../FirebaseRTC/public/prWaitingRoom.html">
```
Comment (Steve-Leke) [inline templates/patWaitingRoom.html:24]: All of the refactoring looks good, however we need to make sure that the files all end up under the public directory and we no longer reference file pathways with ".." or "../.." etc. Check out this link: https://github.com/mellychat/webapp/projects

For the ideal file setup and structure. I don't think we're using the "/temp" folder or the "/FirebaseRTC" folder

Comment (Steve-Leke) [reply templates/patWaitingRoom.html:24]: Lines 25 - 29, 153, 155, 158-159, 162-169 have this issue.

File: FirebaseRTC/public/patWaitingRoom.html [hunk 2]
```diff
@@ -22,11 +22,11 @@
```
Comment (lillianchou94) [inline FirebaseRTC/public/patWaitingRoom.html:22]: does this file need to be deleted? It looks like it's moved to templates/patWaitingRoom.html

File: templates/patWaitingRoom.html (hunk 1 above)
Comment (lillianchou94) [reply templates/patWaitingRoom.html:24]: Let's wait for [this PR](https://github.com/mellychat/webapp/pull/23/files) to merge (contains the temp/assets folder move) then we can update ../temp/assets/... to /assets/...

**Review by lillianchou94 — DISMISSED** [review]

**Review by j0n3lson — APPROVED** [review]

**Pull request merged**